### DISCUSSION TITLE
What happens to tickets with multiple assignees when migrated to the free plan?

### DISCUSSION THREAD

**Discussion by lfrancis** [community/community#23773] [New to GitHub]
<p>What happens to private repo tickets with multiple assignees when an origanization is migrated to the free plan? Do we end up with tickets w/o any assignees? Thanks.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/lfrancis">@lfrancis</a>, Thank you for being here and my apologies for the delayed response. The free plan for organization accounts now comes with private repositories and unlimited users. If your organization no longer require the <a href="https://help.github.com/en/github/getting-started-with-github/faq-about-changes-to-githubs-plans#if-i-downgrade-from-github-team-or-a-legacy-plan-to-github-free-what-features-will-i-lose">additional paid features</a>, you can simply downgrade to the <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/downgrading-your-github-subscription#downgrading-an-organization-with-per-seat-pricing-to-free">free plan</a>.</p>

**Comment by lfrancis** [comment, marked as answer]
<p>Thanks, but my question had to do with how tickets with multiple assignees were handled when a private repo containing such was migrated to the new free plan.</p>
<p>In case this helps anyone with the same question, we ended up migrating anyway and found that tickets maintain their multiple assignments until someone either removes or adds a new assignment, which clears the original assignments.</p>

**Comment by AndreaGriffiths11** [comment]
<aside class="quote no-group quote-modified" data-username="lfrancis" data-post="3" data-topic="18567">
<div class="title">
<div class="quote-controls"></div>
 lfrancis:</div>
<blockquote>
<p>We ended up migrating anyway and found that tickets maintain their multiple assignments until someone either removes or adds a new assignment, which clears the original assignments.</p>
</blockquote>
</aside>
<p>Thanks so much for coming back to share! I’m sure this would help clear this up for others. Thanks again. We’re looking forward to seeing you around!</p>
